libraries/Microsoft.Bot.Builder.AI.LUIS/LuisTelemetryConstants.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    /// <summary>
+    /// The IBotTelemetryClient event and property names that logged by default.
+    /// </summary>
+    public static class LuisTelemetryConstants
+    {
+        public const string LuisResult = "LuisResult";  // Event name
```

Comment:
could these be static readonly? (i.e. do you ever use then in a switch-case statement?)